index.html

```diff
@@ -4297,6 +4297,43 @@ <h2 id="examples-tzigla">Tzigla</h2>
 
     <h2 id="changelog">Change Log</h2>
 
+    <b class="header">1.2.4</b> &mdash; <small><i>Jan. 9, 2016</i></small>
+    &mdash; <a href="https://github.com/jashkenas/backbone/compare/1.2.3...1.2.4">Diff</a>
+    &mdash; <a href="https://cdn.rawgit.com/jashkenas/backbone/1.2.4/index.html">Docs</a>
+    <br />
+    <ul style="margin-top: 5px;">
+      <li>
+        Fixed <tt>_removeModels</tt> regression which could cause an infinite loop
+        under certain conditions.
+      </li>
+      <li>
+        Add <tt>findIndex</tt> and <tt>findLastIndex</tt> Underscore methods to
+        <tt>Collection</tt>.
+      </li>
+      <li>
+        Fixed <tt>Module#id</tt> regression where <tt>id</tt> would be
+        unintentionally `undefined`.
+      </li>
+      <li>
+        <tt>Collection#_removeModels</tt> always returns an array.
+      </li>
+      <li>
+        Fixed <tt>Collection#_onModelEvent</tt> regression where triggering a
+        <tt>change</tt> event without a <tt>model</tt> would error.
+      </li>
+      <li>
+        Ensure <tt>Collection#reduce</tt> and  <tt>Collection#reduceRight</tt>
+        work without an initial <tt>accumulator</tt> value.
+      </li>
+      <li>
+        Fixed <tt>Collection#set</tt> regression when <tt>parse</tt> returns a
+        falsy value.
+      </li>
+      <li>
+        Remove <tt>component</tt> package support.
```

Comment:
No guessing necessary, it's all right there. But sure, we could do that.

On Sat, Jan 9, 2016 at 2:19 PM John-David Dalton <notifications@github.com>
wrote:

> In index.html
> <https://github.com/jashkenas/backbone/pull/3921#discussion_r49267246>:
>
> > +        <tt>Collection#_removeModels</tt> always returns an array.
> > +      </li>
> > +      <li>
> > +        Fixed <tt>Collection#_onModelEvent</tt> regression where triggering a
> > +        <tt>change</tt> event without a <tt>model</tt> would error.
> > +      </li>
> > +      <li>
> > +        Ensure <tt>Collection#reduce</tt> and  <tt>Collection#reduceRight</tt>
> > +        work without an initial <tt>accumulator</tt> value.
> > +      </li>
> > +      <li>
> > +        Fixed <tt>Collection#set</tt> regression when <tt>parse</tt> returns a
> > +        falsy value.
> > +      </li>
> > +      <li>
> > +        Remove <tt>component</tt> package support.
>
> Instead of be trying to guess what you'd like. You could merge and then
> tweak to taste.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jashkenas/backbone/pull/3921/files#r49267246>.
>
-- 

Adam K (mobile)